conf/application.conf

```diff
@@ -249,4 +249,9 @@ mixed-use {
   enabled = true
 }
 
+user-recruitment-banner {
+  enabled = true
```

Comment:
do we need to set this to false in prod for now or are we ok with enabling it in prod straight away?

Reply:
as i understand it there is a push to get this live asap

Reply:
just confirmed with Ashleigh - the sooner the better and the link is good to go now